.github/workflows/nightly.yml

```diff
@@ -0,0 +1,146 @@
+---
+name: 🌙 Nightly Builder
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  check_date:
+    name: 🧐 Check latest commit
+    runs-on: ubuntu-latest
+    outputs:
+      should_run: ${{ steps.should_run.outputs.SHOULD_RUN }}
+
+    steps:
+      - name: ⬇ Checkout pynab
+        uses: actions/checkout@v2
+        with:
+          repository: nabaztag2018/pynab
```

Comment:
Rather than specifying explicity `repository: nabaztag2018/pynab` for checkout actions in this workflow, wouldn't it be better to leave it as default, so that workflow may also work as-is in forked repositories?

Reply:
I'm agreed, but I would recommend to set by the default value `${{ github.repository }}` allowing to overwrite more easily the value in wip fork branches and better understand things.

Reply:
As you like, as long as explicit reference to `nabaztag2018/pynab` is removed.